CHANGELOG.md

```diff
@@ -1,7 +1,29 @@
-## 15.5.0 (April 7, 2017)
+## 15.5.4 (April 11, 2017)
+
+### React Test Renderer
+* Fix compatibility with Enzyme by exposing `batchedUpdates` on shallow renderer. ([@gaearon](https://github.com/gaearon) in [#9382](https://github.com/facebook/react/commit/69933e25c37cf5453a9ef132177241203ee8d2fd).
+
+
+## 15.5.3 (April 7, 2017)
```

Comment:
AFAIK versions before 15.5.4 are using prop-types < 15.5.7 in the UMD, thus having the same [critical issue](https://github.com/reactjs/prop-types/pull/40#issuecomment-297998076). So we should make it clear all of them are effectively deprecated, and people should skip 15.5.0 to 15.5.3, and use at least 15.5.4.